fixtures/ios_app_with_framework_and_resources/StaticFramework3/Derived/Sources/Bundle+StaticFramework3.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
```

Comment:
@kwridan should we gitignore all the files under `Derived`?

Reply:
I guess this will be up to each project to decide, it's ok either way.

They have been useful checked it as when we make changes and re-generate a fixture we can easily see in the git diff if the changes we made somehow produced different results.